src/py42/sdk/__init__.py

```diff
@@ -39,8 +40,8 @@
 
 def from_local_account(host_address, username, password, totp=None):
     """Creates a :class:`~py42.sdk.SDKClient` object for accessing the Code42 REST APIs using the
-    supplied credentials. Currently, only accounts created within the Code42 console or using the
-    APIs (including py42) are supported. Username/passwords that are based on Active Directory,
+    supplied credentials. This method supports only accounts created within the Code42 console or using the
+    APIs (including py42). Username/passwords that are based on Active Directory,
```

Comment:
Should we alter the last sentence here to say

```
Username/passwords that are based on Active Directory, 
Okta, or other Identity providers should use the `from_jwt_provider()` method
```

Reply:
changed.